core/src/avm2/globals/flash/geom/Matrix3D.as

```diff
@@ -584,3 +588,11 @@ package flash.geom {
 
 	}
 }
+
+import flash.geom.Orientation3D;
+
+function checkOrientation(orientationStyle:String) {
+	if (!(orientationStyle == Orientation3D.AXIS_ANGLE || orientationStyle == Orientation3D.EULER_ANGLES || orientationStyle == Orientation3D.QUATERNION)) {
+		throw new Error("Error #2187: Invalid orientation style badOrientation.  Value must be one of 'Orientation3D.EULER_ANGLES', 'Orientation3D.AXIS_ANGLE', or 'Orientation3D.QUATERNION'.", 2187);
```

Comment:
Instead of using the string `badOrientation` this should use `orientationStyle`:
```
throw new Error("Error #2187: Invalid orientation style " + orientationStyle + ".  Value must be one of 'Orientation3D.EULER_ANGLES', 'Orientation3D.AXIS_ANGLE', or 'Orientation3D.QUATERNION'.", 2187);
```